recipes/google-cloud-cpp-compute/run_test_feature.bat

```diff
@@ -0,0 +1,19 @@
+@echo on
+setlocal EnableDelayedExpansion
+
+:: CMake does not like paths with \ characters
+set LIBRARY_PREFIX="%LIBRARY_PREFIX:\=/%"
+
+set FEATURE=%PKG_NAME:libgoogle-cloud-=%
+set FEATURE=%FEATURE:-devel=%
```

Comment:
Windows batch variable syntax is not very self-explanatory; I'd prefer a comment à la
```suggestion
:: strip off prefix and (possible) suffix around the main name of the library (==feature)
set FEATURE=%PKG_NAME:libgoogle-cloud-=%
set FEATURE=%FEATURE:-devel=%
```

Reply:
Removed for this feedstock.  Will fix on the feedstocks where it is relevant.